in_toto/model.go

```diff
@@ -843,3 +851,37 @@ func (mb *Metablock) Sign(key Key) error {
 	mb.Signatures = append(mb.Signatures, newSignature)
 	return nil
 }
+
+/*
+SslSigner provides signature generation and validation based on the SSL
+Signing Spec: https://github.com/secure-systems-lab/signing-spec
+as describe by: https://github.com/MarkLodato/ITE/tree/media-type/ITE/5
+It wraps the generic SSL envelope signer and enforces the correct payload
+type both during signature generation and validation.
+*/
+type SslSigner struct {
```

Comment:
nitpick: I think it's fine to use SSLSigner instead of SslSigner. For example the standard library uses OSCPResponse instead of OscpResponse.

Reply:
Or are you afraid this could be mistaken with SSL/TLS?

Reply:
Same for PAE btw :)

Reply:
> Or are you afraid this could be mistaken with SSL/TLS?

This needn't be a concern, as we're working on renaming the singing-spec. https://github.com/secure-systems-lab/signing-spec/issues/16

Reply:
Yeah, SSL would be more in line, and as I think I mentioned somewhere in the PR "SSL" is a slightly overloaded term so a change would be welcomed :)